Xamarin.Forms.Controls/CoreGalleryPages/LabelCoreGalleryPage.cs

```diff
@@ -9,21 +9,21 @@ protected override bool SupportsFocus
 			get { return false; }
 		}
 
-		protected override void InitializeElement (Label element)
```

Comment:
I think that is too much unrelated change. What about you remove this changes and make it cleaner to the team review it?